src/content/cloud/container-resources.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: Container Resources
```

Comment:
I think the title need some SEO optimization. I don't see my self searching for this when my environment is crashing because of memory. 

Also, why container vs 'dev environments' or something else? We don't use containers in any other element of the sidebar.

Reply:
Do you have a suggestion? This link can be also linked from the app is the `requests` are being exhausted in your dev environment